test/check-application

```diff
@@ -840,7 +840,7 @@ class TestApplication(testlib.MachineCase):
         b.click(alpine_sel + " button.btn-delete")
         self.confirm_modal("Delete")
         self.confirm_modal("Force delete")
-        b.wait_not_in_text("#containers-images", alpine_sel)
+        b.wait_not_in_text("#containers-images", IMG_ALPINE_LATEST)
```

Comment:
Thanks! There another one at https://github.com/cockpit-project/cockpit-podman/blob/main/test/check-application#L807 and  I was expecting it to be more like `b.wait_not_present(alpine_sel)` because the row shouldn't be present at all.

Reply:
Thanks, fixed that as well. And the original fix was broken too, as we have both system and user images.